samples/deployment/test-network/README.md

```diff
@@ -130,27 +131,9 @@ cd $FPC_PATH/samples/deployment/test-network
 ./update-connection.sh
 ```
 
-### How to use simple-go
-
-Now we will use the go app in `$FPC_PATH/samples/application/simple-go` to demonstrate the usage of the FPC Client SDK.
-In order to initiate the FPC Chaincode enclave and register it with the FPC Enclave Registry, run the app with the `-withLifecycleInitEnclave` flag.
-
-```bash
-cd $FPC_PATH/samples/application/simple-go
-CC_ID=echo ORG_NAME=Org1 go run . -withLifecycleInitEnclave
-```
-Note that we execute the go app as `Org1`, thereby creating and registering the FPC Chaincode enclave at `peer0.org1.example.com`. Alternatively, we could run this as `Org2` to initiate the enclave at `peer0.org2.example.com`.
-
-Afterwards you _must_ run the application without the `withLifecycleInitEnclave` flag and you can play with multiple organizations.
-```bash
-cd $FPC_PATH/samples/application/simple-go
-CC_ID=echo ORG_NAME=Org1 go run .
-CC_ID=echo ORG_NAME=Org2 go run .
-```
-
```

Comment:
I believe in order to fix this tutorial issue, we need to touch `$FPC_PATH/samples/application/simple-go` as well. 

I would prefer not removing this, and create a separate PR.

Reply:
I've created PR #660 to fix this part of the tutorial.

Reply:
sure I'll remove that commit

Reply:
I also squashed the commits.
It should be ready.